test/checkpoint/test_checkpoint.c

```diff
@@ -277,7 +282,7 @@ onint(int signo)
 {
 	WT_UNUSED(signo);
 
-	(void)cleanup();
+	testutil_clean_work_dir(home);
```

Comment:
Why switch this from the test-specific ```cleanup```? That function is used to cleanup elsewhere, it's not doing work we need before exit, but I prefer consistency.